vignettes/datatable-benchmarking.Rmd

```diff
@@ -2,12 +2,13 @@
 title: "Benchmarking data.table"
 date: "`r Sys.Date()`"
 output:
-  rmarkdown::html_vignette:
-    toc: true
-    number_sections: true
+  markdown::html_format:
+    options:
+      toc: true
+      number_sections: true
```

Comment:
If you want to style the TOC similarly to **rmarkdown**, you can provide the styles in an `extra.css` file, e.g.,

```css
#TOC {
  border: 1px solid #ccc;
  border-radius: 5px;
  padding-left: 1em;
  background: #f6f6f6;
}
```

Then include this `extra.css` via:

```suggestion
  markdown::html_format:
    options:
      toc: true
      number_sections: true
    meta:
      css: [default, extra.css]
```